packages/mdc-card/_mixins.scss

```diff
@@ -34,6 +34,13 @@
 //
 
 @mixin mdc-card-core-styles($query: mdc-feature-all()) {
+  @include mdc-card-without-ripple($query);
```

Comment:
it looks like card was not already including `mdc-ripple-common` is that intentional?

Reply:
That was probably technically an oversight that went unnoticed because users were already using something else that ended up including it (e.g. button).